prow/config/jobs.go

```diff
@@ -110,6 +110,9 @@ type Presubmit struct {
 
 	UtilityConfig
 
+	// The GitHub team id that is authorized to trigger this job.
+	AuthorizedTeam int `json:"authorized_team"`
```

Comment:
I don't think this belongs in prowjobs. Presubmits should be unrestricted by default and a seperate config should restrict them by job name. We don't need to store this in every job.